bloom/generators/debian/templates/rules.em

```diff
@@ -30,7 +30,8 @@ override_dh_auto_configure:
 	dh_auto_configure -- \
 		-DCATKIN_BUILD_BINARY_PACKAGE="1" \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
-		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)"
+		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
+		-DCMAKE_BUILD_TYPE=RelWithDebInfo
```

Comment:
This is likely to have an impact on our packages. Currently we don't set `Release`, so `RelWithDebInfo` may do more than just add debug info. See: https://github.com/ros-infrastructure/bloom/issues/327